### PR TITLE
fix: add image building workload on tag creation

### PR DESCRIPTION
**Pull request by onedr0p**
This requires the maintainer setting the DOCKERHUB_USERNAME and DOCKERHUB_PASSWORD secrets in the workflow settings on this repo.

This also adds multiarch support for amd64 and arm64 images.

### CODE REVIEW

File: .github/workflows/image.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: image
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
```
Comment (sentriz) [inline .github/workflows/image.yml:6]: what's the `workflow_dispatch:` for?

**Comment by onedr0p** [comment]
Thanks for updating :)

**Review by sentriz — COMMENTED** [review]
looks good with a question. pushed a commit to make the workflow files consistent

File: .github/workflows/image.yml (hunk 1 above)
Comment (onedr0p) [reply .github/workflows/image.yml:6]: that allows you to manually trigger the pipeline
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Comment (sentriz) [reply .github/workflows/image.yml:6]: cool 😎

**Comment by sentriz** [comment]
thanks! updated the password secret to DOCKERHUB_TOKEN because I added a dockerhub access token

hopefully there'll be a release coming soon (after #108) (including some fixes, listenbrainz, podcasts) where we can test this

**Comment by onedr0p** [comment]
Awesome, looking forward to it. Feel free to create a tag for `v0.11.2` to test beforehand :)

**Comment by sentriz** [comment]
Hey @onedr0p  I just did a new release with the new image workflow and it went pretty well 👍
There was one problem with the GITHUB_REF image tag not evaluating, so I added a workaround for that

also the arm build didn't work but that's almost definitely gonics fault

Do you know is there a nice way to run the buildx locally and target arm?

Thanks!

**Comment by onedr0p** [comment]
> Do you know is there a nice way to run the buildx locally and target arm?

I was never able to get that working, I ended up just creating a repo to test things out when messing with it :)

**Comment by onedr0p** [comment]
Thanks for the quick release, already got a PR open for the helm chart

https://github.com/k8s-at-home/charts/pull/541

**Comment by onedr0p** [comment]
I created an issue here to discuss further 

https://github.com/sentriz/gonic/issues/116

**Pull request merged**